DuckDuckGo/AppDelegate.swift

```diff
@@ -903,22 +908,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return window?.rootViewController as? MainViewController
     }
 
-    func refreshShortcuts() {
+    @MainActor
+    func refreshShortcuts() async {
 #if NETWORK_PROTECTION
         guard vpnFeatureVisibility.shouldShowVPNShortcut() else {
             UIApplication.shared.shortcutItems = nil
             return
         }
```

Comment:
This check is redundant now. I kept it for the time being since it's a bit painful to remove or update to do the entitlement check itself, but open to pushback on that.